tests/script-based-pre/playback_array/playback_array.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -e
+set -o pipefail
+set -o nounset
+
+cleanup()
+{
+  rm ${RS_FILE}
+}
+trap cleanup EXIT
+
+RS_FILE="modified.rs"
+cp array.rs ${RS_FILE}
+
+echo "[TEST] Generate test..."
+kani ${RS_FILE} -Z concrete-playback --concrete-playback=inplace
```

Comment:
```suggestion
kani ${RS_FILE} -Z concrete-playback --concrete-playback=inplace || true
```
as this Kani command will have a non-zero return code as it does find property violations (and the `set -e` now makes the script terminate early).

Reply:
Hm, I wonder if we will also need this after the `playback` command, since that fails too?

Reply:
I'd say for the last one it is the right overall exit code.